src/ja/common/message/worker_commands/cancel_job.py

```diff
@@ -9,17 +9,27 @@
 
 class CancelJobCommand(WorkerCommand):
 
+    def __init__(self, job_uid: str):
+        """!
+        @param job_uid: The UID of the job to cancel.
+        """
+        self._job_uid = job_uid
+
     def execute(self, worker_client: JobWorker) -> WorkerResponse:
         """!
         Cancel the job on the worker machine using the provided @worker_client
         @param worker_client:  the Worker object to use for the execution
         @return: a WorkerResponse with the appropriate response
         """
+        return worker_client.docker.cancel_job(self._job_uid)
```

Comment:
The DockerInterface does **not** create Response objects; its methods do not return anything.
The execute method needs to create these Response objects itself.

Reply:
I was planning on using the docker.errors.APIError to ckeck if the command was successful or not. Bc the execute method can't be sure.

Reply:
That sounds like a good solution.